WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -600,12 +600,23 @@ final class EditableOrderViewModel: ObservableObject {
                                        removeProductIntent: { [weak self] in
                 self?.removeItemFromOrder(item)})
         } else if let product = allProducts.first(where: { $0.productID == item.productID }) {
+            let childProductRows = childItems.compactMap { childItem in
+                // If the parent product is a bundle product, quantity cannot be changed.
+                let canChangeQuantity: Bool = {
+                    guard let parentProduct = allProducts.first(where: { $0.productID == item.productID }) else {
+                        return true
+                    }
+                    return parentProduct.productType != .bundle
+                }()
+                return createProductRowViewModel(for: childItem, canChangeQuantity: canChangeQuantity)
+            }
```

Comment:
nit: I think we can assume `childItems` here have `product` as their parent product, how about checking the `canChangeQuantity` for child items by checking the `product`?

```suggestion
            // If the parent product is a bundle product, quantity cannot be changed.
            let canChildItemsChangeQuantity = product.productType != .bundle
            let childProductRows = childItems.compactMap { childItem in
                return createProductRowViewModel(for: childItem, canChangeQuantity: canChildItemsChangeQuantity)
            }
```

Reply:
Ha, yes, I meant to simplify this after I moved it here; thanks!